cmake/install/FindThrust.cmake

```diff
@@ -1,33 +1,28 @@
-
 # FindThrust
 # ---------
 #
 # Try to find Thrust
 #
-# Uses Thrust_ROOT in the cache variables or in the environment as a hint
-# where to search
+# Uses Thrust_ROOT in the cache variables or in the environment as a hint where to search
 #
 # IMPORTED Targets
 # ^^^^^^^^^^^^^^^^
 #
-# This module defines a basic `thrust_create_target` function as provided
-# otherwise by the newer Thrust >= 1.9.10 included configs.
+# This module defines a basic `thrust_create_target` function as provided otherwise by the newer
+# Thrust >= 1.9.10 included configs.
 #
 # Result Variables
 # ^^^^^^^^^^^^^^^^
 #
 # This module defines the following variables:
 #
-# ``Thrust_FOUND``
-#   system has Thrust
-# ``Thrust_INCLUDE_DIRS``
-#   the Thrust include directories
+# ``Thrust_FOUND`` system has Thrust ``Thrust_INCLUDE_DIRS`` the Thrust include directories
```

Comment:
The auto-formatter doesn't do such a great job here.

Reply:
Can fix this with a formatting fence